### PR TITLE
Add header section - home page #15

### PR DESCRIPTION
**Pull request by hamzah-bahar**
### The header section was added: 
- Big  screen:
image
- Medium screen:  
image

- Small screen: 
image
    - When burger icon is pressed: 
image
    - When search icon is pressed: 
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                <h3>Cart</h3>
+            </div>
+            <div class="search-icon">
+                <img src="images/Search icon.png" alt="search icon">
+            </div>
+        </div> 
+        <div class="search-bar hide-search-bar">
+            <label for="search">Search products by name</label>
+            <div class="search">
+                <img src="images/Search icon.png" alt="search icon">
```
Comment (bayanseder) [inline index.html:87]: you can move the icon to the right and make it submit button for the search bar

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This app is platform where you can sell or buy stuff">
+    <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">  
+    <link rel="stylesheet" href="styles/home.css" type="text/css">
+    <title>GITShop</title>
+</head>
+<body> 
+    <!-- Menu for small devices -->
```
Comment (bayanseder) [inline index.html:14]: try to move the menu more to the left (no space between the menu and the edge)

File: index.html (hunk 1 above)
Comment (hamzah-bahar) [reply index.html:87]: There is a form with with input of type submit , so Enter will submit (no need for submit button) 
My plan is to show the results as the user start typing ( if I could 🤞)

File: index.html (hunk 2 above)
Comment (hamzah-bahar) [reply index.html:14]: Ok, you got it

File: index.html (hunk 1 above)
Comment (bayanseder) [reply index.html:87]: you can :+1:

**Review by Rahaf-96 — APPROVED** [review]

**Review by bayanseder — APPROVED** [review]

**Review by saharAdem — APPROVED** [review]

**Pull request merged**